### DISCUSSION TITLE
Substitution filter: <Edit here> as string

### DISCUSSION THREAD

**Discussion by Pierre-de-la-Verre** [WinMerge/winmerge#1962] [General]
I'm fighting with Substitution filter...
Is it possible that the "hint" <Edit here> in column "Replace with" is used as a defined string? So "<Edit here>" gives other results as "" (empty field)?

A little bit confused ...

**Comment by sdottaka** [comment]
If you leave "`<Edit here>`" as is, that string will be used. If you want to replace it with empty, just remove "<Edit here>" and make it empty.

**Comment by Pierre-de-la-Verre** [comment]
Thanks.
